core/trino-main/src/test/java/io/trino/server/ui/TestWebUi.java

```diff
@@ -772,6 +811,24 @@ private void assertAuth2Authentication(HttpServerInfo httpServerInfo, String acc
         assertResponseCode(client, getLocation(baseUri, "/ui/unknown"), SC_NOT_FOUND);
         assertResponseCode(client, getLocation(baseUri, "/ui/api/unknown"), SC_NOT_FOUND);
 
+        if (refreshTokensEnabled) {
```

Comment:
nit: extract to a method?

Reply:
Extracted because now I use this part in two places